e2e/nomostest/new.go

```diff
@@ -337,6 +344,19 @@ func FreshTestEnv(t nomostesting.NTB, opts *ntopts.New) *NT {
 		nt.gitPrivateKeyPath = downloadSSHKey(nt)
 	}
 
+	nt.NotificationServer = &NotificationServer{}
+	if err := nt.NotificationServer.install(nt); err != nil {
+		t.Fatalf("failed to install notification server: %v", err)
+	}
+	if err := nt.NotificationServer.portForward(nt); err != nil {
+		t.Fatalf("failed to port-forward notification server: %v", err)
+	}
+	t.Cleanup(func() {
+		if err := nt.NotificationServer.uninstall(nt); err != nil {
+			t.Errorf("failed to uninstall notification server: %v", err)
+		}
+	})
```

Comment:
Should we install the test-notification-server for all tests?
Can we just install for the notification test specifically?

Reply:
Yeah I was thinking it might make sense to have an option for `nomostest.New` such as `WithInstallNotificationServer` or something similar

Reply:
Done